src/main/java/com/checkmarx/jenkins/CxScanBuilder.java

```diff
@@ -2651,7 +2662,7 @@ public ListBoxModel doFillPostScanActionIdItems(@QueryParameter final boolean us
                                                         @QueryParameter final String username, @QueryParameter final String password,
                                                         @QueryParameter final String timestamp, @QueryParameter final String credentialsId,
                                                         @QueryParameter final boolean isProxy, @AncestorInPath Item item) {
-            Jenkins.getInstance().checkPermission(Item.CONFIGURE);
+                item.checkPermission(Item.CONFIGURE);
```

Comment:
```suggestion
                if (item == null) {
                        return FormValidation.ok();
                }
                item.checkPermission(Item.CONFIGURE);
```
Please check if the context is not null